config/mailinglist/rules/jdk.json

```diff
@@ -12,17 +12,21 @@
         ],
         "client": [
             "make/\\w+(Demos|X11)",
-            "make/autoconf/lib-(font|freetype|x11)",
+            "make/autoconf/lib-(alsa|cups|font|freetype|x11)",
```

Comment:
Is it fine to use in a few places the "font" instead of "fontconfig" and "x11" instead of "x11wrappergen"?

Reply:
My understanding is that the regexp is used to match a prefix. If you add a slash at the end, you enforce a full directory name instead of a partial match. Feel free to rewrite as you like.